schema/validate.go

```diff
@@ -53,6 +74,28 @@ func main() {
 		for _, desc := range result.Errors() {
 			fmt.Printf("- %s\n", desc)
 		}
+	}
+
+	return nil
+}
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "validate"
+	app.Usage = usage
+
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "schema,s",
+			Value: "",
+			Usage: "specify the schema-json file",
```

Comment:
There's no validation without a schema, so I'd rather keep this as a positional parameter (and not make it an option).